bin/vitamin.js

```diff
@@ -147,8 +146,8 @@ const build = (options, hotCallback, restartServer) => (options.hot ?
 
 
 const test = (args) => {
-    npmRunSpawn(
-        'jest',
+    spawn(
+        vitaminResolve('node_modules', '.bin', 'jest'),
```

Comment:
> `node_modules/.bin/jest`
The problem is that jest can be installed inside vitaminJS node_modules (in the case of a link, or when the user also installed jest with another version), or inside the app node_modules (in case of standard installation).

If we call `node_modules/.bin/jest` we should resolve "à la" npm, in order to prevent issues with that.
That's exactly what `npmRun.spawn` does.